src/pipelinedefinition.cpp

```diff
@@ -228,6 +228,15 @@ Status PipelineDefinition::waitForLoaded(std::unique_ptr<PipelineDefinitionUnloa
     return StatusCode::OK;
 }
 
+static bool useSharedOutputContent(const tensorflow::serving::PredictRequest* request){
+    return true;
+}
+
+static bool useSharedOutputContent(const ::inference::ModelInferRequest* request){
+    return request->raw_input_contents().size() > 0;
+}
```

Comment:
Move this out of pipeline definition. Probably to tfs/kfs_utils? Then resuse this function in modelinstance.